includes/embeds/class-amp-vimeo-embed.php

```diff
@@ -50,7 +67,9 @@ public function __construct( $args = [] ) {
 	 */
 	public function register_embed() {
 		wp_embed_register_handler( 'amp-vimeo', self::URL_PATTERN, [ $this, 'oembed' ], -1 );
-		add_shortcode( 'vimeo', [ $this, 'shortcode' ] );
+		if ( ! $this->is_amp_shortcode_available_in_jetpack( self::SHORTCODE_TAG ) ) {
```

Comment:
See notes above for Soundcloud.

Reply:
Sure, https://github.com/ampproject/amp-wp/pull/3678/commits/b64392bf469255d7573e074a8bc0d0d1f1dc14c8 changes this